app/models/content_item.rb

```diff
@@ -171,12 +171,16 @@ def router_rendering_app
     rendering_app || "government-frontend"
   end
 
-  def viewable_by?(user_uid)
-    authorised_user_uids.empty? || authorised_user_uids.include?(user_uid)
+  def viewable?(bypass_id: nil, user_id: nil, organisation_id: nil)
+    return auth_bypass_ids.include?(bypass_id) if bypass_id
+    return user_id.present? if auth_bypass_ids.any?
```

Comment:
I don't think we need this line - when I first saw it it raised a concern to me of "what if user_id is 'invalid'" (this is what prompted me to suggest removing the "invalid" values in controller). Then thinking about it a bit more I don't think we care whether user is authenticated or not, as we expect that to be already handled by auth proxy.